### ISSUE TITLE
Duplicate headers issue with latest release

### ISSUE THREAD

**Issue by arwineap** [pivotal-cf/email-resource#34]
It looks like with today's release, there seem to be some header issues.

Originally I used the headers filed as described in the README.md
```
MIME-version: 1.0
Content-Type: text/html; charset="UTF-8"
```

However, I'm now running into errors looking like this:
```
v1.0.15-d5ab14181de9a8f66a5bed57e60da6adc7a212b8 - Error during execute - Error closing:: 554 Transaction failed: Duplicate header 'Content-Type'.
```
And after removing content-type:
```
v1.0.15-d5ab14181de9a8f66a5bed57e60da6adc7a212b8 - Error during execute - Error closing:: 554 Transaction failed: Duplicate header 'MIME-Version'.
```

Once I remove both of them, it's able to send email and is setting these headers for the values:
```
Content-Type: multipart/mixed; boundary="0d545cae4a990f55ba95086fc85790d23a2deb3fd787767402526f53e99d"; charset=UTF-8
Mime-Version: 1.0
```

It seems like it's related to this commit:
https://github.com/pivotal-cf/email-resource/commit/8bdfdd1d71ef35ef66fcafbbaeecde68cdb9b7c8

The header processing looks fine (in out.go), but then I think something goes wrong in the sender.go additions.

Maybe it's getting added a second time here?
https://github.com/pivotal-cf/email-resource/blob/master/out/sender.go#L69-L70

**Comment by calebwashburn** [comment]
I'll update to ensure there is a better way to send html email but for now you can move from `latest` tag to `1.0.14` which backs this change out.  I'll update latest shortly to fix this issue.

**Comment by calebwashburn** [comment]
Fixed with `1.0.16` which is also tagged as `latest` now

**Comment by arwineap** [comment]
Thanks!
